app/src/main/java/org/oppia/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -22,6 +21,7 @@ class ProfileEditViewModel @Inject constructor(
   private val profileManagementController: ProfileManagementController
 ) : ObservableViewModel() {
   private lateinit var profileId: ProfileId
+  private lateinit var switch: Switch
```

Comment:
Ditto here (per later comment--please add TODO since this may leak memory).

Reply:
Done!